README.md

```diff
@@ -1,7 +1,8 @@
 # Seismometer
```

Comment:
This looks great, thank you so much for making this update.

We want to make sure we acknowledge efforts, but we don't have anything automated in place just yet...  
So as an early (the **first** non-maintainer) contributor, I'll pose it as a question, how would you like to be acknowledged? Is there more you'd want to see than the automatic Github Contributors card?

Reply:
Thanks for asking! Nothing else is needed on my end.

Looking forward to helping with other issues down the road.